controller/tap/apiserver_test.go

```diff
@@ -21,24 +21,24 @@ func TestNewAPIServer(t *testing.T) {
 		err    error
 	}{
 		{
-			err: errors.New("failed to load [extension-apiserver-authentication] config: configmaps \"extension-apiserver-authentication\" not found"),
+			err: fmt.Errorf("failed to load [%s] config: configmaps %q not found", k8sutils.ExtensionAPIServerAuthenticationConfigMapName, k8sutils.ExtensionAPIServerAuthenticationConfigMapName),
 		},
 		{
 			err: nil,
 			k8sRes: []string{`
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: extension-apiserver-authentication
-  namespace: kube-system
+ name: extension-apiserver-authentication
```

Comment:
Can we change these into 2-space indentation?

Reply:
Yes definitely. Sorry about that.